### ISSUE TITLE
`DTEND` should be optional in `VTEVENT`

### ISSUE THREAD

**Issue by chrisdotcode** [JaninaWibker/parse-ical#2]
Hi there,

I'm parsing some iCal data with some `VTEVENT`s, yet no `DTEND`s. The spec official says that [`DTEND` is *optional* on `VTEVENT`s](https://www.rfc-editor.org/rfc/rfc5545#section-3.6.1), but currently, the code mandates it.

- - -
Spec Reference:
```EBNF
rrule /
;
; Either 'dtend' or 'duration' MAY appear in
; a 'eventprop', but 'dtend' and 'duration'
; MUST NOT occur in the same 'eventprop'.
;
dtend / duration /
```

The corresponding code is here:
https://github.com/JaninaWibker/parse-ical/blob/main/src/transform.ts#L134

Thanks!

**Comment by chrisdotcode** [comment]
Along with that, from a quick peruse of the code, I don't necessarily see the rule that `DTEND` is currently exclusive with `DURATION` at the moment.

**Comment by JaninaWibker** [comment]
This should be fixed in `parse-ical@1.2.0`, but please do test it out and report back if it works correctly 😅

**Comment by chrisdotcode** [comment]
@JaninaWibker Seems to work! Thanks!
